### PR TITLE
New version: Igoravl.TfsCmdlets version 2.1.4+2594.1

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/36478)

### CODE REVIEW

File: manifests/i/Igoravl/TfsCmdlets/2.1.4+2594.1/Igoravl.TfsCmdlets.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Igoravl.TfsCmdlets
+PackageVersion: 2.1.4+2594.1
```
Comment (OfficialEsco) [inline manifests/i/Igoravl/TfsCmdlets/2.1.4+2594.1/Igoravl.TfsCmdlets.installer.yaml:5]:
```suggestion
PackageVersion: 2.1.4.2594
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
```
Status: Completed
##[error] Result: Failed
##[error] Manifest is invalid. (Line: 10, Col: 1, Idx: 302) - (Line: 10, Col: 2, Idx: 303): While parsing a block mapping, did not find expected key.
##[error]Manifest Schema Validation Failed

****************************************************************************************
Manifest Schema Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed
##[error] One or more errors occurred. (Manifest is invalid.) Manifest is invalid. (Line: 10, Col: 1, Idx: 302) - (Line: 10, Col: 2, Idx: 303): While parsing a block mapping, did not find expected key.
Finishing
```

**Comment by vedantmgoyal2009** [comment]
The error is occurring in release notes probably. Can you tell how to fix? I have tried removing certain characters which I think were causing error.

**Comment by OfficialEsco** [comment]
Well, it does say its on line 10, which is broken in the Installer Yaml.

its missing InstallModes on line 10.

File: manifests/i/Igoravl/TfsCmdlets/2.1.4.2594/Igoravl.TfsCmdlets.locale.en-US.yaml [hunk 2]
```diff
@@ -27,6 +28,17 @@ Tags:
...
+ReleaseNotes: |
+Version 2.1.4 (30/Nov/2021)
+
+This release fixes a couple issues in Get-TfsIdentity and Invoke-TfsRestApi.
+
+Fixes
 
+- Error when using Get-TfsIdentity with Azure DevOps Services- Get-TfsIdentity is a cmdlet that helps to deal with legacy APIs, such as the Azure DevOps Security API. To fetch an identiy, it connected to a configuration server (the "root" of an Azure DevOps installation) and called the required API. However, connecting to a configuration server is not supported when using Azure DevOps Services. Now, when used with Azure DevOps Services, Get-TfsIdentity will use the collection (organization) scope for the API, whereas calls to the on-prem server still target the configuration server.
+
+- Error calling Invoke-TfsRestApi for alternate hosts under Windows PowerShell (Desktop)- There was a bug in the implementation of Invoke-TfsRestApi that caused it to fail when using alternate hosts in Windows PowerShell. This has been fixed.
```
Comment (OfficialEsco) [inline manifests/i/Igoravl/TfsCmdlets/2.1.4.2594/Igoravl.TfsCmdlets.locale.en-US.yaml:41]:
```suggestion
  Version 2.1.4 (30/Nov/2021)

  This release fixes a couple issues in Get-TfsIdentity and Invoke-TfsRestApi.

  Fixes

  - Error when using Get-TfsIdentity with Azure DevOps Services- Get-TfsIdentity is a cmdlet that helps to deal with legacy APIs, such as the Azure DevOps Security API. To fetch an identiy, it connected to a configuration server (the "root" of an Azure DevOps installation) and called the required API. However, connecting to a configuration server is not supported when using Azure DevOps Services. Now, when used with Azure DevOps Services, Get-TfsIdentity will use the collection (organization) scope for the API, whereas calls to the on-prem server still target the configuration server.

  - Error calling Invoke-TfsRestApi for alternate hosts under Windows PowerShell (Desktop)- There was a bug in the implementation of Invoke-TfsRestApi that caused it to fail when using alternate hosts in Windows PowerShell. This has been fixed.
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**